src/jade/tabs/trust.jade

```diff
@@ -84,24 +84,16 @@ section.col-xs-12.content(ng-controller="TrustCtrl")
                     rp-autofill-counterparty_view
                     rp-autofill-on='addform_visible = true'
                     rp-focus, rp-spinner="{{counterparty_loading ? 4 : null}}")
-                  .errorGroup(rp-errors='trust_counterparty')
-                    div(rp-error-valid)
+                  .errorGroup(ng-messages='trustForm.trust_counterparty.$error', ng-if='trustForm.trust_counterparty.$dirty')
+                    div.ng-valid
                       .success(ng-show='counterparty != counterparty_address && counterparty_name && !error_account_reserve')
                         | {{counterparty_address}}
                       .error(ng-show='error_account_reserve')
                         span(l10n) This account does not meet the minimum XRP reserve, so you cannot use it as a gateway.
-                    .error(rp-error-on='required', l10n) Please enter a Ripple name, contact, or address.
-                    .error(rp-error-on='rpNotMe', l10n) You've entered your own address.
-                    .error(rp-error-on='rpDest', l10n) Please enter a valid Ripple name, contact, or address.
-                    .error(rp-error-on='rpAccountExists', l10n) You cannot add an unfunded Ripple account as a gateway.
-              .errorGroup(rp-errors='trust_amount')
-                .error(rp-error-on='required', l10n) Amount field is required
-              .errorGroup(rp-errors='trust_amount_currency')
-                .error(rp-error-on='rpNotXrp', l10n)
-                  | XRP can be sent without extending trust. If you really want
-                  |  to trust in XRP, please use XRR currency code.
-                .error(rp-error-on='required', l10n)
-                  | Currency field is required
```

Comment:
@yongsoo I remove L97-L104 because this shouldn't be here (a duplicate)? The same errorGroup's already exist below.